libs/application/form/src/components/Screen.tsx

```diff
@@ -92,38 +76,22 @@ const Screen: FC<ScreenProps> = ({
       // call submit mutation
       console.log('here we will submit', formValue)
     } else {
-      if (applicationId) {
-        await updateApplication({
-          variables: {
-            input: {
-              id: applicationId,
-              typeId: formTypeId,
-              answers: data,
-            },
-          },
-        })
-      } else {
-        await createApplication({
-          variables: {
-            input: {
-              applicant: '123456-1234',
-              state: 'PENDING',
-              attachments: {},
-              typeId: formTypeId,
-              assignee: '123456-1235',
-              externalId: 'some_id',
-              answers: data,
-            },
+      await updateApplication({
```

Comment:
Not related to this refactor, but will we need to handle if this has an error?

Reply:
yes! we really need to do some error handling. I will add that in a later PR. Hopefully there are some error message components or something similar in the ui kit.